packages/www/components/Dashboard/WebhooksTable/index.tsx

```diff
@@ -118,6 +129,32 @@ const WebhooksTable = ({ title = "Webhooks" }: { title?: string }) => {
                 cursor: "pointer",
               },
             },
+            status: {
+              children: (
+                <Box>
+                  {!webhook.status ? (
+                    <StatusBadge
+                      variant={StatusVariant.Idle}
+                      tooltipText="This webhook hasn't received any events yet"
+                    />
+                  ) : webhook.status.lastFailure &&
+                    +Date.now() - webhook.status.lastFailure.timestamp <
+                      WARNING_TIMEFRAME ? (
+                    <StatusBadge
+                      variant={StatusVariant.Unhealthy}
+                      timestamp={webhook.status.lastTriggeredAt}
+                      tooltipText="A trigger has failed in the last hour. Last triggered"
```

Comment:
Followup on my other comment, I think this would be cleaner and give the most useful information:

```suggestion
                      timestamp={webhook.status.lastFailure.timestamp}
                      tooltipText="Last error"
```

I think tooltips should ideally be really short.